package.json

```diff
@@ -4,32 +4,45 @@
   "description": "An H5P widget that sets the value of the field to a [UUID](https://en.wikipedia.org/wiki/Universally_unique_identifier). The widget does not render the input field.",
   "main": "h5p-editor-uuid.js",
   "scripts": {
-    "start": "vite build --mode=development --watch --minify=false",
-    "build": "tsc && vite build",
-    "lint": "prettier .",
-    "lint:fix": "prettier . --write"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "NODE_ENV=production webpack",
+    "watch": "webpack --watch",
+    "lint": "eslint \"./**/*.ts\"",
+    "lint:fix": "eslint --fix \"./**/*.ts\""
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/NDLA-H5P/h5p-editor-uuid.git"
+    "url": "git+https://github.com/NDLANO/h5p-editor-uuid.git"
   },
-  "keywords": [],
-  "author": "",
-  "license": "ISC",
+  "keywords": [
+    "h5p",
+    "library"
+  ],
+  "author": "NDLA",
+  "license": "MIT",
   "bugs": {
-    "url": "https://github.com/NDLA-H5P/h5p-editor-uuid/issues"
+    "url": "https://github.com/NDLANO/h5p-editor-uuid/issues"
   },
-  "homepage": "https://github.com/NDLA-H5P/h5p-editor-uuid#readme",
+  "homepage": "https://github.com/NDLANO/h5p-editor-uuid#readme",
   "devDependencies": {
     "@tsconfig/strictest": "^2.0.1",
-    "h5p-types": "^1.17.0",
-    "prettier": "^2.8.8",
-    "prettier-config": "github:boyum/prettier-config",
-    "typescript": "^5.0.4",
-    "vite": "^4.3.3"
+    "@typescript-eslint/eslint-plugin": "^5.59.2",
+    "eslint": "^8.39.0",
+    "eslint-config-standard-with-typescript": "^34.0.1",
```

Comment:
What are `eslint-config-standard-with-typescript`, `eslint-plugin-import`, `eslint-plugin-n` and `eslint-plugin-promise` required for? One could use `eslint-config-standard-with-typescript`, sure, but it's not configured anywhere and here it does nothing.